fr.nytuo.cosmiccomics.yml

```diff
@@ -0,0 +1,64 @@
+app-id: fr.nytuo.cosmiccomics
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=host
```

Comment:
This is pretty broad. Why do you need it? Electron supports using the filechooser portal.

Reply:
I only use electron as a client for the front end of my application, to give the user a pleasant desktop experience.
The application is mainly intended to be hosted on a server with both a backend and/or a frontend server.
The Electron version run the server as a background child process and the frontend from local files.
So, I don't rely on electron's exclusive features such as the showOpenDialog.

Reply:
so what the whole point of giving host access?

Reply:
The server must have access to the file system in order to read files from the path specified by the user in the frontend. The fact that the file system is defined as host is necessary in cases where the user has his files stored outside his home directory (such as an external disk mounted in /mnt or /media, for example). Giving the user the right to use only the home directory is of course possible, but the user cannot access files outside this directory.